drivers/display/display_st7789v.c

```diff
@@ -409,14 +409,14 @@ static void st7789v_enter_sleep(struct st7789v_data *data)
 }
 
 static int st7789v_pm_control(const struct device *dev, uint32_t ctrl_command,
-				 void *context, device_pm_cb cb, void *arg)
+				 uint32_t *state, device_pm_cb cb, void *arg)
```

Comment:
that's a nice improvement. I wonder if states could be represented by an `enum`.

Reply:
yep, I will work on states like we did to system power management. Just changed this because this signature was driving me crazy, every time I looked it I thought that context was some private data passing across callbacks but was just the state :/